client/mds3/test/mds3.integration.spec.js

```diff
@@ -209,12 +200,23 @@ tape('Custom dataset with custom variants, WITH samples', test => {
 		}
 		//Confirm number of custom variants matches in block instance
 		test.equal(
-			mds3Track.custom_variants.length,
+			tk.custom_variants.length,
 			variantNum.size,
 			`Should render total # of custom variants = ${variantNum.size}`
 		)
 
-		if (test._ok) holder.remove()
+		const firstM = tk.skewer.g.selectAll('.sja_aa_disckick').nodes()[0]
+		// click firstM
+		firstM.dispatchEvent(new Event('click'))
+		await whenVisible(tk.itemtip.d)
+		test.pass('itemtip shows after clicking first m')
+
+		const span = await detectOne({ elem: tk.itemtip.dnode, selector: '.pp_mds3_singleSampleNameSpan' })
+		console.log(span)
+		test.ok(span, 'pp_mds3_singleSampleNameSpan <span> found')
+		test.equal(span.innerHTML, custom_variants[0].sample, 'correct sample name shows in <span>')
```

Comment:
@creilly8 it shows innerHTML is undefined, do you know how to access the value?

Reply:
@xzhou82 It's not the `innerHTML` that is undefined, its `custom_variants[0].sample`. The sample property is replaced by a samples array (somewhere in the block code, maybe?). I tried `custom_variants[0].samples[0].sample_id` in `test.equal()` and it worked.